lib/extensions/ar_adapter/ar_pglogical.rb

```diff
@@ -0,0 +1,228 @@
+module PgLogical
+  def pglogical_installed?
+    exec_query("SELECT EXISTS(SELECT * FROM pg_available_extensions WHERE name = 'pglogical')").cast_values.first
+  end
+
+  # Enables pglogical postgres extensions
+  def pglogical_enable
+    enable_extension("pglogical")
+    enable_extension("pglogical_origin")
+  end
+
+  # Node Management
+  #
+
+  # Creates a node
+  #
+  # @param name [String]
+  # @param dsn [String] external connection string to the node
+  def pglogical_node_create(name, dsn)
+    raw_connection.exec_params("SELECT pglogical.create_node($1::VARCHAR, $2::VARCHAR)", [name, dsn])
```

Comment:
Oh, huh. Constructing an Attribute just for that seems rather pointless, so I think `raw_connection.exec_params` (or `raw_connection.async_exec`, actually?) is probably the way to go.